test/hotspot/jtreg/serviceability/jdwp/DebuggeeLauncher.java

```diff
@@ -107,12 +104,7 @@ public int getJdwpPort() {
 
     @Override
     public void onStringRead(StreamHandler handler, String line) {
-        if (handler.equals(errorHandler)) {
-            terminateDebuggee();
-            listener.onDebuggeeError(line);
-        } else {
-            processDebuggeeOutput(line);
-        }
+        processDebuggeeOutput(line);
```

Comment:
So the fix is not to terminate the debuggee and raise an error if
there is output on stderr? The line will simply be processed just
like any line that occurs on stdout?

If my understanding is correct, then I'm good with this fix.

Reply:
Not exactly, the stderr is just ignored. We don't expect anything so just ignore it.

Reply:
So previously we would terminate the debuggee the first time we saw any output on stderr, and now instead we ignore stderr.  What if instead we process stderr the same as stdout? Would that cause a failure because the output is not what the test expects? What if there really is an issue launching the JVM that is expressed on stderr? Wouldn't we want that detected by this code?

Reply:
Agree, lets process stderr like stdout. This shouldn't change logic of test but allow us at least to see stderr.

Reply:
I'm not seeing how calling `processDebuggeeOutput` lets us "see" stderr? If there is nothing the test needs to read from stderr then I don't see any point in processing it.

IIUC this change will impact a lot of tests - correct? If so this needs a lot of testing - more than the original change that passed on the VM flags which did not catch this problem. Maybe that change should be backed out until we have a clearer idea of how to handle this extraneous output issue more broadly?

Reply:
> I'm not seeing how calling processDebuggeeOutput lets us "see" stderr? If there is nothing the test needs to read from stderr then I don't see any point in processing it.

I was thinking that maybe processDebuggeeOutput() should be updated to log all the output, maybe with "debuggee stderr: " and "debuggee stdout: " prefixes on each line.